codecept.conf.js

```diff
@@ -65,6 +65,7 @@ exports.config = {
     notifyGatekeeperEventPage: './e2e/pages/events/sendCaseToGatekeeperEvent.page.js',
     createNoticeOfProceedingsEventPage: './e2e/pages/events/createNoticeOfProceedingsEvent.page.js',
     addHearingBookingDetailsEventPage: './e2e/pages/events/addHearingBookingDetailsEvent.page.js',
```

Comment:
Going to remove this?

Reply:
Just seen that there is a ticket to clean stuff up, this going to be part of it?

Reply:
In FPLA-2280 will cleanup